address-space-controller/src/main/java/io/enmasse/controller/common/KubernetesHelper.java

```diff
@@ -434,12 +437,12 @@ public void createServiceAccount(String namespace, String saName) {
     public void addAddressSpaceRoleBindings(AddressSpace addressSpace) {
         String namespace = addressSpace.getAnnotation(AnnotationKeys.NAMESPACE);
 
-        if (isRBACSupported() && client.isAdaptable(OpenShiftClient.class)) {
+        if (isRBACSupported()) {
             createRoleBinding("address-space-viewers", namespace, "ClusterRole", "view", Arrays.asList(
-                    new Subject("ServiceAccount", "default", namespace)));;
+                    new Subject("ServiceAccount", "default", namespace)));
             createRoleBinding("address-admins", namespace, "ClusterRole", "edit", Arrays.asList(
                     new Subject("ServiceAccount", addressSpaceAdminSa, namespace)));
-            if (hasClusterRole("event-reporter")) {
+            if (client.isAdaptable(OpenShiftClient.class) && hasClusterRole("event-reporter")) {
```

Comment:
@lulf I've assumed here that the "event-reporter" role is OpenShift specific. Is that correct?

Reply:
@calohmn I think we can actually report the whole if sentence

Reply:
@lulf you mean remove the "if" line? On my minikube cluster, there is no "event-reporter" ClusterRole, though. Or do you mean remove the whole "if" block?